package.json

```diff
@@ -3,12 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.2",
```

Comment:
Is this needed? I'd rather not depend on material-ui as status components are significantly different

Reply:
It was being used in the components `Welcome.jsx` and `Tooltip.jsx`. Hence added. I can definitely remove it.

Reply:
Please! Those components are not used, just copy pasta leftovers :D

Reply:
Sure 👍